lib/statsd/instrument/strict.rb

```diff
@@ -111,31 +107,113 @@ def collect_metric(type, name, value, sample_rate:, tags: nil, prefix:, metadata
     end
 
     module StrictMetaprogramming
-      def statsd_measure(method, name, sample_rate: nil, tags: nil,
-        prefix: StatsD.prefix, no_prefix: false, as_dist: false)
-
+      def statsd_measure(method, name, sample_rate: nil, tags: nil, no_prefix: false)
         check_method_and_metric_name(method, name)
-        super
+
+        # Unfortunately, we have to inline the new method implementation ebcause we have to fix the
```

Comment:
```suggestion
        # Unfortunately, we have to inline the new method implementation because we have to fix the
```

this typo was also copied to following methods